### ISSUE TITLE
Cannot handle non-rectangular coordinate systems - Runtime Error

### ISSUE THREAD

**Issue by gkb999** [ZiluM/sacpy#2]
Hi there, 
I'm trying to plot in orthographic/stereographic mode, but it raises a runtime error : 
![image](https://user-images.githubusercontent.com/7091088/234414132-c924745c-3de4-4a2e-b776-c4055b5238a2.png)

RuntimeError: Cannot handle non-rectangular coordinate systems.
############
Is it that sacpy only supports rectangular plots? Or am I not doing it right?

Thanks in advance

**Comment by ZiluM** [comment]
Yes! Sacpy is built under Cartopy Support and Cartopy can't support this setting ticks. I am sorry for that.
I am trying to support this using other methods. 
Zilu Meng
















At 2023-04-26 06:02:36, "GK" ***@***.***> wrote:

Hi there,
I'm trying to plot in orthographic/stereographic mode, but it raises a runtime error :


RuntimeError: Cannot handle non-rectangular coordinate systems.
############
Is it that sacpy only supports rectangular plots? Or am I not doing it right?

Thanks in advance

**Comment by gkb999** [comment]
Thanks for getting back and yeah, I have tried using other methods, somewhat within sacpy, this isn't possible.
Thanks for keeping it clear and please let me know if you're successful with other methods you're trying to support with.
Also, there's a function in linger_cal.py that defines multi-corr and on partial_corr, how does the partial_corr work?
Can you please provide links to any documents?
